### PR TITLE
Handle windows paths in workspace.get_uri_like + test

### PR DESCRIPTION
**Pull request by tomv564**
Fix for #85 where windows paths were not prefixed and converted to forward slashes

### CODE REVIEW

**Review by gatesn — COMMENTED** [review]
Good catches here. One thing I'd say, is do we need to url quote the path we use in the uri_like function? I know it's not part of this Pr but may as well slip it in

**Comment by tomv564** [comment]
Sounds like a good idea, I'll add quoting. 
It will try to (unnecessarily) encode the `:` in `/D:/myfile` which I'd like to avoid.
I couldn't find any reason why module_path is sometimes None, please let me know if my assumption was correct!

File: pyls/workspace.py [hunk 1]
```diff
@@ -181,5 +181,10 @@ def get_uri_like(doc_uri, path):
     unicode objects.
     """
     parts = list(urlparse(doc_uri))
+    if path[0] != '/':  # fix path for windows
```
Comment (gatesn) [inline pyls/workspace.py]: Can we add `and ':' in path` just to make sure we don't throw on the next line

**Review by gatesn — APPROVED** [review]
LGTM

**Pull request merged**